test/packageTypeScript.test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect } from 'chai';
 
-import StrategyDefaultImport, { Strategy as StrategyNamedImport } from '..';
```

Comment:
thought: I don't like this change to be honest, since this way we don't get a sanity-check on resolution defined in `package.json`. I'd rather disable the problemtic ESlint import rule.